doc/ADMIN.md

```diff
@@ -0,0 +1,10 @@
+## Limitations
+
+* By default, all users have access to the accounts. The admin needs to manually select the authorised user through YunoHost Administration Panel.
+
+## Roadmap
+
+* to be added:
+  * [ ] URL cannot be changed
+  * [ ] Improve log file and add logrotate
```

Comment:
Are those points to be removed?

Reply:
Hm, I guess I missed it while rebasing. I'll edit in on testing, good catch!

Reply:
Fixed! Thanks